lib/Frontend/Frontend.cpp

```diff
@@ -353,10 +370,6 @@ bool CompilerInstance::setUpModuleLoaders() {
     Context->addModuleLoader(std::move(MemoryBufferLoader));
   }
 
-  std::unique_ptr<SerializedModuleLoader> SML =
-    SerializedModuleLoader::create(*Context, getDependencyTracker(), MLM);
-  this->SML = SML.get();
```

Comment:
Huh, I wonder if this order mattered at one point.

Reply:
Even if it did, I don't see a reason to keep it because it doesn't seem to matter now :).